pkg/apis/stork/v1alpha1/migration.go

```diff
@@ -21,6 +21,7 @@ type MigrationSpec struct {
 	StartApplications            *bool             `json:"startApplications"`
 	PurgeDeletedResources        *bool             `json:"purgeDeletedResources"`
 	SkipServiceUpdate            *bool             `json:"skipServiceUpdate"`
+	SkipNetworkPolicyCheck       *bool             `json:"skipNetworkPolicyCheck"`
```

Comment:
may be we can rename this  to `includeNetworkPolicyWithCIDR` ? Goes well with other configs like `includeResources`. `SkipNetworkPolicyCheck` doesn't indicate what check is being called out in the Migration object

Reply:
agree, renamed it to `includeNetworkPolicyWithCIDR`